kf_lib_data_ingest/target_apis/cached_schema.json

```diff
@@ -0,0 +1,1343 @@
+                    "type":"string",
+                    "x-nullable":true
+                },
+                "created_at":{
+                    "description":"Time of object creation",
+                    "format":"date-time",
+                    "readOnly":true,
+                    "type":"string",
+                    "x-nullable":true
+                },
+                "data_access_authority":{
+                    "type":"string"
+                },
+                "external_id":{
+                    "description":"dbGaP accession number",
+                    "type":"string"
+                },
+                "investigator_id":{
+                    "example":"IG_ABB2C104",
+                    "maxLength":11,
+                    "type":"string",
+                    "x-nullable":true
+                },
+                "kf_id":{
+                    "description":"ID assigned by Kids First",
+                    "maxLength":11,
+                    "type":"string"
+                },
+                "modified_at":{
+                    "description":"Time of last modification",
+                    "format":"date-time",
+                    "readOnly":true,
+                    "type":"string",
+                    "x-nullable":true
+                },
+                "name":{
+                    "description":"Name or title of study",
+                    "type":"string",
+                    "x-nullable":true
+                },
+                "release_status":{
+                    "description":"Release status of the study",
+                    "enum":[
+                        "Publishing",
+                        "Running",
+                        "Canceled",
+                        "Published",
+                        "Waiting",
+                        "Not Available",
+                        "Not Applicable",
+                        "Failed",
+                        "Staged",
+                        "Pending",
+                        "Not Reported",
+                        "Reported Unknown",
+                        "Not Allowed To Collect"
+                    ],
+                    "type":"string",
+                    "x-nullable":true
+                },
+                "short_name":{
+                    "description":"Short name for study",
+                    "type":"string",
+                    "x-nullable":true
+                },
+                "version":{
+                    "description":"dbGaP version",
+                    "type":"string",
+                    "x-nullable":true
+                },
+                "visible":{
+                    "description":"Flags visibility of data from the dataservice",
+                    "type":"boolean"
+                }
+            },
+            "required":[
+                "external_id"
+            ],
+            "type":"object"
+        }
+    },
+    "version":{
```

Comment:
I think its a good idea to version control the kf data service schema content + version along with this library (or really, wherever the concept_schema is). This way we can tell which version of the data service schema is being used in the standard to target concept mapping in kids_first.py